### PR TITLE
Overview of my own series #6938

### LINKED ISSUE #6938: Overview of my own series
**Describe the solution you'd like**

In the personal menu, near "Dashboard", there should be a "My Series" item, leading to an overview of series I created. Alternatively, dashboard could get "All" (like now), "Posts", "Series", and "Comments" tabs.

The "Series: …" indicator above articles in the dashboard should link to an overview page for that series (cf. #2424).

*Other users* could get a link to the “all series by…” overview on my profile page.

### PR DESCRIPTION
**Pull request by Oghenebrume50**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR attempts to address the issue #6938 this user proposed a series overview on the dashboard, I had come up with a workable solution, so I opened this draft PR so I can get more suggestion on what exactly is needed for this to be perfect

Closes #6938

## QA Instructions, Screenshots, Recordings
This is how it shows when the dashboard is opened
series1

This is how it shows when the series is clicked on, basically just list all series if any, clicking on any one links to the first article in the series
series2

since all other articles in the articles are shown here I think this works.
series3

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
@rhymes @ludwiczakpawel I was opening when I saw @ludwiczakpawel comment on the issue #6938  if that is the way to go then fine I will refactor, but I wanted to discuss things  like design structure, should the series just show as a list as it is now with maybe more separation or are there other things that will be needed alongside each series :slightly_smiling_face:

**Comment by ludwiczakpawel** [comment]
I'll wait with reviewing the PR until we're on the same page in https://github.com/thepracticaldev/dev.to/issues/6938

**Comment by Oghenebrume50** [comment]
so apparently the merge of  #9201 has taken care of a lot of things with regards to #6938 and  #2424 the series overview issue, all that was left to do is to make the series page accessible from the dashboard, so I modified this PR to do just that.

see

if this is not the right place to put it due to the changes overtime, I can fix that 

@rhymes @ludwiczakpawel

**Comment by Oghenebrume50** [comment]
I am not sure why Travis is failing, any insight with regards to that would be greatly appreciated

**Comment by rhymes** [comment]
@Oghenebrume50 not sure either, I've restarted it :D

**Comment by Oghenebrume50** [comment]
Passes now, that's a relief!!!

File: app/controllers/dashboards_controller.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ def show
     @articles = @articles.sorting(params[:sort]).decorate
     @articles = Kaminari.paginate_array(@articles).page(params[:page]).per(50)
 
+    @collections_count = @user.collections.count
```
Comment (citizen428) [inline app/controllers/dashboards_controller.rb:32]: **question (non-blocking):** Is `@collections_count` used anywhere? The view here uses `@user.collections.count` too.

Comment (Oghenebrume50) [reply app/controllers/dashboards_controller.rb]: This was an oversight, I have removed it.

**Comment by Oghenebrume50** [comment]
@mstruve I have corrected the schema, I do  not even know how that got there :D

File: app/views/dashboards/_actions.html.erb [hunk 2]
```diff
@@ -4,6 +4,11 @@
     <span class="crayons-indicator"><%= @user.articles_count %></span>
   </a>
 
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?) %>" href="/<%=current_user.username%>/series">
```
Comment (citizen428) [inline app/views/dashboards/_actions.html.erb:7]: **suggestion (non-blocking):** This line is quited long and hard to read. Maybe something like this instead?

```suggestion
  is_own_series = params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?)
  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if is_own_series %>" href="/<%=current_user.username%>/series">
```

**Pull request merged**